patches/chromium/port_autofill_colors_to_the_color_pipeline.patch

```diff
@@ -0,0 +1,97 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: deepak1556 <hop2deep@gmail.com>
+Date: Mon, 14 Feb 2022 19:24:36 +0900
+Subject: Port autofill colors to the color pipeline.
+
+https://chromium-review.googlesource.com/c/chromium/src/+/1652925 removed colors as they are no longer
+needed in chromium but our autofill implementation uses them.  This patch can be removed if we refactor
+our autofill implementation to work like Chromium's.
```

Comment:
I didn't look into refactoring the autofill implementation, instead ported the `put_back_deleted_colors_for_autofill.patch` to the new color pipeline to get this PR moving.